From mboxrd@z Thu Jan  1 00:00:00 1970
Return-Path: <linux-rockchip-bounces+linux-rockchip=archiver.kernel.org@lists.infradead.org>
X-Spam-Checker-Version: SpamAssassin 3.4.0 (2014-02-07) on
	aws-us-west-2-korg-lkml-1.web.codeaurora.org
Received: from bombadil.infradead.org (bombadil.infradead.org [198.137.202.133])
	(using TLSv1.2 with cipher ECDHE-RSA-AES256-GCM-SHA384 (256/256 bits))
	(No client certificate requested)
	by smtp.lore.kernel.org (Postfix) with ESMTPS id 40E3BD30CC3
	for <linux-rockchip@archiver.kernel.org>; Tue, 13 Jan 2026 20:00:02 +0000 (UTC)
DKIM-Signature: v=1; a=rsa-sha256; q=dns/txt; c=relaxed/relaxed;
	d=lists.infradead.org; s=bombadil.20210309; h=Sender:
	Content-Transfer-Encoding:Content-Type:List-Subscribe:List-Help:List-Post:
	List-Archive:List-Unsubscribe:List-Id:MIME-Version:References:In-Reply-To:
	Message-ID:Date:Subject:Cc:To:From:Reply-To:Content-ID:Content-Description:
	Resent-Date:Resent-From:Resent-Sender:Resent-To:Resent-Cc:Resent-Message-ID:
	List-Owner; bh=yEL2h/pcvMA031ICnkJLawj1rMvuJAl+AJr7iviCE1Q=; b=dsOsbswT8qiN0L
	HEg7iH18opRJ1/L0j+oGF7SEQKeiCfdntORtYwr38Gzo2Wqglgf/c9TlSjeMeMotj7r1WF0l6Yuc8
	iOq8GvXHxtaMPwXy50z8HPY3tfjj3ZWGFyte3ZDMllJLcnWXYMjJJpjeqbplTnhZDEvTWS+YFEauX
	1BNFP/TXrtdFTiiMdE8LZIeziiGKi5L/IPIIIpmqaykg1tM3FvaavRziKafV/7vi7mRbBsFGVbTQS
	ExD5u819wyVau1iXm2X8aKcpd3u3ZcPg1TvC3gP49cGn0DFyLuycSU5r810EhWyLyDQ8AeE4pxT/r
	e8sNiyXcpPd8bciQScpw==;
Received: from localhost ([::1] helo=bombadil.infradead.org)
	by bombadil.infradead.org with esmtp (Exim 4.98.2 #2 (Red Hat Linux))
	id 1vfkYV-00000007eaC-0AGs;
	Tue, 13 Jan 2026 19:59:55 +0000
Received: from mail-ot1-x329.google.com ([2607:f8b0:4864:20::329])
	by bombadil.infradead.org with esmtps (Exim 4.98.2 #2 (Red Hat Linux))
	id 1vfkYQ-00000007eY3-1LIu
	for linux-rockchip@lists.infradead.org;
	Tue, 13 Jan 2026 19:59:51 +0000
Received: by mail-ot1-x329.google.com with SMTP id 46e09a7af769-7c6dbdaced8so6528280a34.1
        for <linux-rockchip@lists.infradead.org>; Tue, 13 Jan 2026 11:59:50 -0800 (PST)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
        d=gmail.com; s=20230601; t=1768334389; x=1768939189; darn=lists.infradead.org;
        h=content-transfer-encoding:mime-version:references:in-reply-to
         :message-id:date:subject:cc:to:from:from:to:cc:subject:date
         :message-id:reply-to;
        bh=SqMQUQ/aJfEqDI2GLO0SzFgPGkoYzlQYbdw+E86b2Os=;
        b=Xx7ipNzHzzeSPBZskFbHlFXr2mvf/SO0twYdzoENbmQR8W75dfZnxgQrhpgnQC1C1d
         pzMqm1Kv/FIwSRNvZC9PTdcdYmPT+971M1OiVy0sPFE0+NWdbRaftSWStVHw2QeUBKn/
         gotw3i8pFwcTfXyD5XyH02vbP+jcGH5b84QZ++7DAiyzVntQ4H+96GPAFLiuOp/+FruG
         itcdoZHnBHADTt5rD0Pecrlc6aRzBj5cBJhEM5KrcKM6M/0tOQdLVn7KPIm1fIqgcmVC
         tw6v65Dq0Wa9kyqNgT5pnknLo1565clrAvl1cN9v19DNjI0eqMQvwxuAI6dJFLjx6L47
         nexA==
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
        d=1e100.net; s=20230601; t=1768334389; x=1768939189;
        h=content-transfer-encoding:mime-version:references:in-reply-to
         :message-id:date:subject:cc:to:from:x-gm-gg:x-gm-message-state:from
         :to:cc:subject:date:message-id:reply-to;
        bh=SqMQUQ/aJfEqDI2GLO0SzFgPGkoYzlQYbdw+E86b2Os=;
        b=D4hUJAwV7qmc8oglcfu0cIdoVVdBDwUhRtUkF9a4HYRlTRk1X2DUm22Oj5q+O6h4BP
         L0dLqFSskyqmikNkrjaQPiNgkeNK4gDfW0YFkUwNH++M6RWUJkbeyVnv/r4UsbwN1BXl
         48xSKaLAYKAZdPsSLUBvVQWskiebLVp4baGSuohHG86HeQVzpJiientiqorqeSNk+u9O
         Az4txOuji1XIc9Tz8Ry2ws1iqoZLt8ugF50APU1TgAJFNl1ymApHg72F6MbW6t+YbCaI
         j5GCqUjgtvoZVgZUEUXSP8Y6dDd4tukjU7BdUl5NL0m8az2oY4Pztk4MOJbbQNTzEB6x
         G5xA==
X-Gm-Message-State: AOJu0YxkQitDfKEsX9KiVhvX1CVB5qBJRHVB76Y+ih0PoeKRspZ8wOYh
	cYg8uoWhjTnNxLq+OfmZTEgznGgGo5W7OdTHWPNzP3X7a2bEq9YtLnbQS6Nd5w==
X-Gm-Gg: AY/fxX54WPxJ7iCZtAVtyRXwCMJuJ4ZSduOFnfbAaK63ReFELlX1BDIERQXhBxw66Cs
	u4Gpdn9bq9SNY/JA3vFJ1ClCBGOqenW/SQsiIMxDUMxaXirsZmlOGEgvApGQ/bpNGVpqdOHy2BZ
	Aaz6gmy8lWConRTca5dReUJ6PXEn4pQ2Cla3uJgzrBZzvhw8/ZTttmEyCUpG/NxAcv1Q8UJbeOt
	Zti0Sn9C3BwezBIy1AZiMFHPuVq6drxO4C02U8dYDcDnMsAVVNJ5mNPCeHSoF3LXKXBQsOGAioh
	HDpErdduza0vUrMai51l+RB8ctfrufCeRe3DNDtkW8cQKTSIrKJu+Tqy/RnXbl7bziYVt4njJN8
	e5PRrFZ5rMUUKUCi2znCsrPLVG8k5+KfWxCo3hz3SiSG8KzTjc2oPw+Krp5XSm2osTL06FcHbEP
	A8isd/tCzh+977MsmsLuc=
X-Received: by 2002:a05:6830:4ac6:b0:7c7:701:fb10 with SMTP id 46e09a7af769-7cfc8a40079mr209141a34.7.1768334389066;
        Tue, 13 Jan 2026 11:59:49 -0800 (PST)
Received: from localhost.localdomain ([2600:1700:fb0:1bc0::54])
        by smtp.gmail.com with ESMTPSA id 46e09a7af769-7ce47801d4dsm16221119a34.5.2026.01.13.11.59.48
        (version=TLS1_3 cipher=TLS_AES_256_GCM_SHA384 bits=256/256);
        Tue, 13 Jan 2026 11:59:48 -0800 (PST)
From: Chris Morgan <macroalpha82@gmail.com>
To: linux-rockchip@lists.infradead.org
Cc: dri-devel@lists.freedesktop.org,
	devicetree@vger.kernel.org,
	dmitry.torokhov@gmail.com,
	simona@ffwll.ch,
	airlied@gmail.com,
	tzimmermann@suse.de,
	mripard@kernel.org,
	maarten.lankhorst@linux.intel.com,
	jesszhan0024@gmail.com,
	neil.armstrong@linaro.org,
	jagan@edgeble.ai,
	heiko@sntech.de,
	conor+dt@kernel.org,
	krzk+dt@kernel.org,
	robh@kernel.org,
	aweinzerl13@yahoo.com,
	Chris Morgan <macromorgan@hotmail.com>
Subject: [PATCH 2/6] dt-bindings: display: panel: Add compatible for Anbernic RG-DS
Date: Tue, 13 Jan 2026 13:57:17 -0600
Message-ID: <20260113195721.151205-3-macroalpha82@gmail.com>
X-Mailer: git-send-email 2.43.0
In-Reply-To: <20260113195721.151205-1-macroalpha82@gmail.com>
References: <20260113195721.151205-1-macroalpha82@gmail.com>
MIME-Version: 1.0
X-CRM114-Version: 20100106-BlameMichelson ( TRE 0.8.0 (BSD) ) MR-646709E3 
X-CRM114-CacheID: sfid-20260113_115950_380993_AEBAFE20 
X-CRM114-Status: UNSURE (   8.94  )
X-CRM114-Notice: Please train this message.
X-BeenThere: linux-rockchip@lists.infradead.org
X-Mailman-Version: 2.1.34
Precedence: list
List-Id: Upstream kernel work for Rockchip platforms <linux-rockchip.lists.infradead.org>
List-Unsubscribe: <http://lists.infradead.org/mailman/options/linux-rockchip>,
 <mailto:linux-rockchip-request@lists.infradead.org?subject=unsubscribe>
List-Archive: <http://lists.infradead.org/pipermail/linux-rockchip/>
List-Post: <mailto:linux-rockchip@lists.infradead.org>
List-Help: <mailto:linux-rockchip-request@lists.infradead.org?subject=help>
List-Subscribe: <http://lists.infradead.org/mailman/listinfo/linux-rockchip>,
 <mailto:linux-rockchip-request@lists.infradead.org?subject=subscribe>
Content-Type: text/plain; charset="us-ascii"
Content-Transfer-Encoding: 7bit
Sender: "Linux-rockchip" <linux-rockchip-bounces@lists.infradead.org>
Errors-To: linux-rockchip-bounces+linux-rockchip=archiver.kernel.org@lists.infradead.org

From: Chris Morgan <macromorgan@hotmail.com>

The Anbernic RG-DS uses two (mostly) identical panels as a top
and bottom panel which appear to use the same controller as the
Jadard JD9365DA-H3. The panels differ with a parameter defined
differently in the init sequence.

Signed-off-by: Chris Morgan <macromorgan@hotmail.com>
---
 .../devicetree/bindings/display/panel/jadard,jd9365da-h3.yaml   | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Documentation/devicetree/bindings/display/panel/jadard,jd9365da-h3.yaml b/Documentation/devicetree/bindings/display/panel/jadard,jd9365da-h3.yaml
index b8783eba3ddc..5802fb3c9ffe 100644
--- a/Documentation/devicetree/bindings/display/panel/jadard,jd9365da-h3.yaml
+++ b/Documentation/devicetree/bindings/display/panel/jadard,jd9365da-h3.yaml
@@ -16,6 +16,8 @@ properties:
   compatible:
     items:
       - enum:
+          - anbernic,rg-ds-display-bottom
+          - anbernic,rg-ds-display-top
           - chongzhou,cz101b4001
           - kingdisplay,kd101ne3-40ti
           - melfas,lmfbx101117480
-- 
2.43.0


_______________________________________________
Linux-rockchip mailing list
Linux-rockchip@lists.infradead.org
http://lists.infradead.org/mailman/listinfo/linux-rockchip

